coreTable/CoreTableColumns.ts

```diff
@@ -186,7 +187,8 @@ abstract class AbstractCoreColumn<JS_TYPE extends PrimitiveType> {
         return this.formatValue(value)
     }
 
-    formatForTick(value: any, options?: TickFormattingOptions) {
+    // Needs to be bound to instance to pass it to Axis without creating an intermediate function
+    @bind formatForTick(value: any, options?: TickFormattingOptions) {
```

Comment:
nit: I've found `@bind` to be a code smell. I think passing the column around might be a better approach, and now that i hink about it we may be able to utilize having column inside the axis class more, beyond formatting